src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -0,0 +1,234 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using Xunit;
+
+public static class WebSocketTests
+{
+    [Fact]
+    [OuterLoop]
+    [ActiveIssue(424)]
```

Comment:
If new WebSocket packages are available on Monday, verify if these tests still fail and if they don't, remove the ActiveIssue.